scripts/initialize_database.py

```diff
@@ -127,9 +124,9 @@ def run():
 
     if file_exists(db_file_path):
```

Comment:
Rather than `os`, let's use the `pathlib` module:

```
from pathlib import Path

...

def get_db_file_path(sql_uri: str) -> Path:
  ...
  return Path(db_file_path)

if db_file_path.exists():
  ...
```